tests/queries/0_stateless/03221_script_line_number.sql

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
This test seems messy, please add "explanation SELECT" statements like in this other test: tests/queries/0_stateless/02494_query_cache_drop_cache.sql